HidSharp/Platform/Windows/NativeMethods.cs

```diff
@@ -873,7 +873,11 @@ public enum BDIF : uint
             Unknown2         = 0x2000000  // ???
         }
 
-        public static int[] RestrictedHIDUsage = new int[] { 1, 2, 6, 7 };
+        public static Dictionary<int, List<int>> RestrictedHIDs = new Dictionary<int, List<int>>
+        {
+            [0x01] = { 1, 2, 6, 7 },
+            [0x0D] = { 1, 2, 4, 5 }
+        };
```

Comment:
Add a comment indicating the source of this list.

Reply:
Requested change pushed.